docs/content/3.middleware/2.guides/3.extensions.md

```diff
@@ -16,6 +16,7 @@ export const integrations = {
         ...extensions, // don't forget to add existing extensions
         {
           name: 'extension-name',
+          isNamespaced: `[true/false:default]`
```

Comment:
I did not know that there is an "official notation for describing all possibilities of options in a software documentation", could you please share a link? ;)

Reply:
@bartoszherba I did not know either, I thought you did and that's why you used it here :P But it's OK, I was just asking for a friend because he was curious